EventFilter/EcalRawToDigi/src/DCCEBEventBlock.cc

```diff
@@ -164,7 +164,7 @@ void DCCEBEventBlock::unpack(const uint64_t * buffer, size_t numbBytes, unsigned
   
     // Unpack SRP block
     if(srChStatus_ != CH_TIMEOUT &&  srChStatus_ != CH_DISABLED){
-      STATUS = srpBlock_->unpack(&data_,&dwToEnd_);
+      STATUS = srpBlock_->unpack(&data_,&dwToEnd_,SRP_NUMBFLAGS);
```

Comment:
Yes. With the using statement in the header clang now see two signatures of unpack that take two parameters: one is the base class function and the other is the derived class function with one of three parameters defaulted.